wiki/Beatmap_Ranking_Procedure/en.md

```diff
@@ -0,0 +1,43 @@
+# Beatmap Ranking Procedure
+
+Beatmaps can be promoted to the community by following the beatmap ranking procedure and entering the Ranked beatmaps category.
+
+## Feedback
+
+A beatmap's creator can mark their submissions as `Work-in-progress` or `Pending`. Beatmaps in both of these categories are intended to receive feedback from the osu! community.
+
+[Modding](/wiki/Modding) is the process of receiving constructive criticism on a beatmap and is beneficial to improving its quality. Users ordinarily post feedback on a beatmap's discussion page or discuss issues with the beatmap's creator directly.
+
+Newer beatmappers usually need extensive modding to create beatmaps that meet the Ranked category's quality standards, and these beatmappers are often required to re-do their beatmaps from scratch.
+
+Before being nominated for ranking, beatmapsets need to receive [hype](/wiki/Glossary#hype) posts from at least five different users.
+
+## Nominations
+
+Members of the [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) (*BN*) are experienced modders who are capable of nominating beatmapsets for ranking. Their role is to ensure a beatmapset meets the [ranking criteria](/wiki/Ranking_Criteria) and their own personal quality standards.
+
+Members of the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (*NAT*) may also nominate beatmaps, however that is not their primary role.
+
+It is recommended to receive mods before requesting nominations from Beatmap Nominators, however five Hypes are the only pre-requisite.
+
+## Qualification
```

Comment:
This section should probably also mention the 7 day timer and perhaps mention the queue briefly (it is covered to a greater extent in the next section but it comes a bit out of nowhere right now I feel).

Reply:
this is mentioned in the nominatino reset section, since it's only applicable to those maps

Reply:
The fact that the map is put into the Ranked category after 7 days is not only applicable to maps that get resets. The queue is, but not the timer.

Reply:
"at least 7 days" is probably good enough for this section, explain the queue somewhere else